test/1_api.js

```diff
@@ -119,6 +120,7 @@ describe('Server', function() {
 		storage.clear();
 		xhr = sinon.useFakeXMLHttpRequest();
 		clock = sinon.useFakeTimers();
+		server.createScript = function() {};
```

Comment:
It's safer. Since we are stubbing it anyway, I'd prefer to nullify it here so that we don't have to worry about it.